assets/scss/_home.scss

```diff
@@ -576,8 +576,38 @@
         }
     }
 
-    #complextopics .container {
-        margin-top: auto;
+    #complextopics {
+        @media (min-width: 1600px) {
+            max-height: 643px;
+        }
+    
+        .container {
+            margin-top: auto;
+        }
+
+        .section-img {
+            img {
+                margin-top: -6em;
+                min-width: 95px;
+                width: 15vw;
```

Comment:
I think the character is more in line with the sizes of the other sections with `width: 17vw;` - other breakpoints are fine.